src/NightwatchServiceProvider.php

```diff
@@ -133,6 +136,12 @@ final class NightwatchServiceProvider extends ServiceProvider
 
     public function register(): void
     {
+        if (! method_exists(Event::class, 'tap')) {
+            Event::macro('tap', function ($callable) {
+                return tap($this, $callable);
+            });
+        }
+
```

Comment:
Do we want to move this to Compatibility class and do a version compare instead of this? The `Tappable` trait was added to `Event` in version 11.x